src/gfx_func.h

```diff
@@ -52,8 +52,10 @@ extern byte _dirkeys;        ///< 1 = left, 2 = up, 4 = right, 8 = down
 extern bool _fullscreen;
 extern byte _support8bpp;
 extern CursorVars _cursor;
-extern bool _ctrl_pressed;   ///< Is Ctrl pressed?
-extern bool _shift_pressed;  ///< Is Shift pressed?
+extern bool _shift_pressed;    ///< Is Shift pressed?
+extern bool _fn_pressed;       ///< Is function modifier pressed?
```

Comment:
I find the naming of this confusing. It has nothing to do with the 'Fn' button that exists on my keyboard.
Although I do not immediately have a better name for it. I'd almost say '_alternate_behaviour_pressed' or something similar, as it's an indication of alternate behaviour. However, that would somewhat conflict with the actual 'Alt' button on the keyboard.